### PR TITLE
Imporove docs and fix diagnostics

### PR DESCRIPTION
**Pull request by guilhermebodin**
@raphaelsaavedra Following the discussion of #114 I checked and all math is equal, now I think it is of no use add the dependency to the project ci, but nice contribution! :) :rocket: 

Anyway, I fixed feel things and improved the LogAP example, there was an error there when naming the components.

### CODE REVIEW

File: docs/src/examples.md [hunk 1]
```diff
@@ -29,32 +29,40 @@ model = structural(logAP, 12)
 Estimating the model gives us the trend and seasonal components of the time series.
 
 ```julia
-# Estimate a StateSpace structure
 ss = statespace(model)
 
-# Analyze its decomposition in trend and seasonal
-p2 = plot(AP[:Date], [ss.smoother.alpha[:, 1] ss.smoother.alpha[:, 3]], layout = (2, 1),
-            label = ["Trend component" "Seasonal component"], legend = :topleft)
+# Analyze its decomposition in level, trend and seasonal
```
Comment (raphaelsaavedra) [inline docs/src/examples.md]: Should be trend, slope, and seasonal

File: docs/src/examples.md [hunk 2]
```diff
@@ -29,32 +29,40 @@ model = structural(logAP, 12)
...
 p3 = plot!(p1, newdates, pred, label = "Forecast")
 ```
 
 ![Forecast for log of Air Passengers](./assets/logap_forecast.png)
 
+Another feature of StateSpaceModels.jl is to simulate scenarios of the time series. In this example
+we will simulate 10 scenarios for 24 months ahead
+
+```julia
+# 10 scenarios for 24 months ahead
```
Comment (raphaelsaavedra) [inline docs/src/examples.md]: Remove commentary for consistency

File: docs/src/examples.md [hunk 3]
```diff
@@ -29,32 +29,40 @@ model = structural(logAP, 12)
...
+firstdate = AP.Date[end] + Month(1)
 newdates = collect(firstdate:Month(1):firstdate + Month(N - 1))
 
 p3 = plot!(p1, newdates, pred, label = "Forecast")
 ```
 
 ![Forecast for log of Air Passengers](./assets/logap_forecast.png)
 
+Another feature of StateSpaceModels.jl is to simulate scenarios of the time series. In this example
+we will simulate 10 scenarios for 24 months ahead
```
Comment (raphaelsaavedra) [inline docs/src/examples.md]:
```suggestion
we will simulate 10 scenarios for 24 months ahead.
```

File: examples/airpassengers.jl [hunk 4]
```diff
@@ -4,27 +4,31 @@ using CSV, StateSpaceModels, Plots, Statistics, Dates
...
 # Specify the state-space model
 model = structural(logAP, 12)
 
 # Estimate the state-space model
 ss = statespace(model)
 
-# Analyze its decomposition in trend and seasonal
-p2 = plot(AP[:Date], [ss.smoother.alpha[:, 1] ss.smoother.alpha[:, 3]], layout = (2, 1),
-            label = ["Trend component" "Seasonal component"], legend = :topleft)
+# Analyze its decomposition in level, trend and seasonal
```
Comment (raphaelsaavedra) [inline examples/airpassengers.jl]: Trend, slope, and seasonal

Comment (guilhermebodin) [reply examples/airpassengers.jl]: Now I think the most consistent is trend as the sum and seasonal 
image

Comment (guilhermebodin) [reply examples/airpassengers.jl]: I see trend as level + slope

Comment (raphaelsaavedra) [reply examples/airpassengers.jl]: But that's exactly it -- μ is the trend, which also includes the slope ν inside it.

So the states are the trend (which includes the slope), the slope, and the seasonal

Comment (raphaelsaavedra) [reply examples/airpassengers.jl]: Sometimes the trend is also referred to as the level but I think it's less clear what it means in that case

Comment (raphaelsaavedra) [reply examples/airpassengers.jl]: However, calling the slope "trend" is much more misleading, so I'd say we leave it as trend/slope/seasonal

Comment (raphaelsaavedra) [reply examples/airpassengers.jl]: Regarding trend == level + slope: 

level + slope is simply the trend at t+1 though. so it's not a different state, and also does not have its own state equation.

**Pull request merged**